mlir/test/Conversion/GPUToSPIRV/reductions.mlir

```diff
@@ -789,3 +789,44 @@ gpu.module @kernels {
   }
 }
 }
+
+// -----
+
+module attributes {
+  gpu.container_module,
+  spirv.target_env = #spirv.target_env<#spirv.vce<v1.3, [Kernel, Addresses, Groups, GroupUniformArithmeticKHR, GroupNonUniformClustered], []>, #spirv.resource_limits<>>
+} {
+
+gpu.module @kernels {
+  // CHECK-LABEL:  spirv.func @test
```

Comment:
This is not testing exactly the name? Also it's better to use a more descriptive name like `cluster_reduce`. (I know existing tests are not doing that though.)